README.md

```diff
@@ -1,60 +1,64 @@
 
 ## Pre-requisites
-1. You have healthcare data (e.g. EHR, claims, lab, HIE, etc.) in a data warehouse
-2. You have [dbt](https://www.getdbt.com/) installed and configured (i.e. connected to your data warehouse)
+1. You have healthcare data (e.g. EHR, claims, lab, HIE, etc.) in a data warehouse.
+2. You have run one of the Tuva Project's connectors or preprocessing projects ([claims_preprocessing_snowflake](https://github.com/tuva-health/claims_preprocessing_snowflake), [claims_preprocessing_redshift](https://github.com/tuva-health/claims_preprocessing_redshift), etc.) to transform your data. 
+Alternatively, you can directly map your data to the source schema described in [source.yml](models/source.yml).
+3. You have [dbt](https://www.getdbt.com/) installed and configured (i.e. connected to your data warehouse).
 
 [Here](https://docs.getdbt.com/dbt-cli/installation) are instructions for installing dbt.
 
-## Configuration
-Execute the following steps to load all seed files, build all data marts, and run all data quality tests in your data warehouse:
+## Getting Started
+Complete the following steps to configure the package to run in your environment.
 
 1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repo to your local machine or environment
-2. This project will run automatically off of [Tuva Core](https://github.com/tuva-health/core).
-3. Alternatively, if you haven't installed Tuva Core you can directly map your data to the source schema described in [source.yml](models/source.yml)
-4. Configure [dbt_project.yml](/dbt_project.yml)
-    - profile: write the name of the profile in the profile.yml file that connects to your data warehouse
-    - vars: configure source_name, source_database name, and source_schema name
-5. Run project
-    1. Navigate to the project directory in the command line
-    2. Execute "dbt build" to create all tables/views in your data warehouse
+2. Configure [dbt_project.yml](/dbt_project.yml)
+    - Profile: set to 'tuva' by default - change this to an active profile in the profile.yml file that connects to your data warehouse 
+    - Fill in vars (variables):
+      - source_name - description of the dataset feeding this project 
+      - input_database - database where sources feeding this project are stored 
+      - input_schema - schema where sources feeding this project is stored 
+      - output_database - database where output of this project should be written. We suggest using the Tuva database but any database will work. 
+      - output_schema - name of the schema where output of this project should be written
+3. Review [sources.yml](models/sources.yml). The table names listed are the same as in the Tuva data model (linked above).  If you decided to rename these tables:
+    - Update table names in sources.yml
+4. Execute `dbt build` to load seed files, run models, and perform tests.
+
+## Usage Example
```

Comment:
@thutuva I know this is different from CPP. I like having this example here for readmissions and chronic_conditions. I think it could help people get started that may not be familiar with using vars in dbt commands.

Reply:
That's smart.  I'll go back and update this on the other repos.